### PR TITLE
fix example indent

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @QuietMisdreavus

### CODE REVIEW

File: src/libcore/fmt/mod.rs [hunk 1]
```diff
@@ -1576,10 +1576,10 @@ impl<'a> Formatter<'a> {
     /// impl fmt::Debug for Foo {
     ///     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
     ///         fmt.debug_struct("Foo")
-    ///             .field("bar", &self.bar)
-    ///             .field("baz", &self.baz)
-    ///             .field("addr", &format_args!("{}", self.addr))
-    ///             .finish()
```
Comment (frewsxcv) [inline src/libcore/fmt/mod.rs]: for what it's worth, this is how rustfmt formats it currently:

https://play.rust-lang.org/?gist=90c28f98c5dc3e5c98b1a4a16d7aee96&version=nightly

(try pressing the 'Format' button)

Comment (GuillaumeGomez) [reply src/libcore/fmt/mod.rs]: Urg. It's not aligned with the first line. What's the point then? :-/

**Comment by QuietMisdreavus** [comment]
The style RFCs have apparently agreed on using block indent, not aligning the dot operators: <https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/expressions.md#chains-of-fields-and-method-calls>

**Comment by GuillaumeGomez** [comment]
Oh come on...

**Comment by topecongiro** [comment]
cc https://github.com/rust-lang-nursery/fmt-rfcs/issues/125.

**Comment by pietroalbini** [comment]
Ping from triage! What's the status of this PR?

**Comment by QuietMisdreavus** [comment]
I'm tempted to close this PR if the linked fmt-rfc issue isn't resolved in its favor. Right now, the PR would cause the example to break style guidelines. I'm not sure whether there are other examples in std that feature this kind of method-chaining, though.

**Comment by frewsxcv** [comment]
I'm in agreement with @QuietMisdreavus. I'm going to close for now

**Pull request closed without merging**